### ISSUE TITLE
High disk write volume on .cache sqlite databases. 

### ISSUE THREAD

**Issue by maxmcd** [denoland/deno#24750]
Version: Deno 1.44.2

We run lots of deno processes that frequently resolve dependencies with the `--reload` flag. Our hosts that run Deno processes will regularly see 300mb/s of write volume to the sqlite databases in the cache. I think the v8 cache is the highest volume although I seem to have misplaced the strace logs I was using to verify this.

Couple questions:
1. Is the source of this high write volume known? These databases are usually 5mb to 80mb so I was surprised to see write volume that exceeded the size of the entire DB. Maybe there is lots of state contention and sqlite is writing lots of data over and over again?
2. Would it be possible to restructure the `.cache` directory so that the sqlite databases are in their own folder so that I can either bind mount or tmpfs that folder and skip the disk entirely? (Maybe this is possible already, but I assumed I cannot somehow make the sqlite files memory backed without including the full directory. 
3. Open to any other ideas to help reduce the disk usage :) 

Thanks!

**Comment by igorzi** [comment]
Are your dependencies changing frequently where `--reload` is forcing recompilation? It's possible that your use case would see no benefit from V8 code cache. Can you try running with `--no-code-cache`.

**Comment by maxmcd** [comment]
@igorzi the sources with `--reload` are typically identical. I also did some local testing with `--no-code-cache` and still saw high write volume to the other sqlite dbs.

**Comment by igorzi** [comment]
Would it be possible to strace to see which specific dbs?

**Comment by maxmcd** [comment]
@igorzi yes, I will try to get a representative production strace, will take a little time.

**Comment by bartlomieju** [comment]
Closing due to lack of response to the information request. Please reopen if you can provide the requested details.
